aten/src/ATen/native/quantized/cpu/qlinear_dynamic.cpp

```diff
@@ -182,17 +153,53 @@ class QLinearDynamicInt8 final : public torch::OperatorKernel {
               /*bias=*/bias_ptr,
               /*nCol=*/N);
 
-      // Do the GEMM
-      fbgemm::fbgemmPacked(
-          /*packA=*/packA,
-          /*packB=*/*packB,
-          /*C=*/output.data_ptr<float>(),
-          /*C_buffer=*/buffer.data_ptr<int32_t>(),
-          /*ldc=*/N,
-          /*outProcess=*/outputProcObj,
-          /*thread_id=*/0,
-          /*num_threads=*/1);
-    }
+          // Do the GEMM
+          fbgemm::fbgemmPacked(
+              /*packA=*/packA,
+              /*packB=*/*packB,
+              /*C=*/output.data_ptr<float>(),
+              /*C_buffer=*/buffer.data_ptr<int32_t>(),
+              /*ldc=*/N,
+              /*outProcess=*/outputProcObj,
+              /*thread_id=*/task_id,
+              /*num_threads=*/num_tasks);
+
+        } else if (pack_ptr.q_scheme == kPerChannelAffine) {
```

Comment:
Does this mean that we support per-channel dynamic quant as a qconfig option?

Reply:
Yes, I think so. We added the per channel quantization support for dynamic quantization a while ago.